### PR TITLE
Few-Shot Learning using Siamese Neural Network (Contrastive Loss Function)

### PR DESCRIPTION
**Pull request by Nathanael349**
This is the few-shot learning model using the Siamese Neural Network Architecture and it is averaging middle 80s to 90s in accuracy and is consistently having a high AUC-ROC score.

### CODE REVIEW

**Review by Nathanael349 — COMMENTED** [review]
This is the Triplet Loss Function, results are similar to the Contrastive Loss Function.

**Review by JustinLin905 — APPROVED** [review]
